queries/runescript/highlights.scm

```diff
@@ -0,0 +1,117 @@
+; Variables
+(identifier) @variable
+
+(local_variable
+  "$" @attribute.builtin
+  name: (identifier) @variable)
+
+(game_variable
+  [
+    "%"
+    ".%"
+  ] @attribute.builtin
+  name: (identifier) @variable)
+
+(constant_variable
+  "^" @attribute.builtin
+  name: (identifier) @variable)
+
```

Comment:
Last thing, can we get

```scm
(parameter
  (local_variable
    name: (identifier) @variable.parameter))
```
here? Making sure it is after the identifier highlight so it overrides it